### DISCUSSION TITLE
Error in v0.52.0-beta4

### DISCUSSION THREAD

**Discussion by ZIKVLD** [rejetto/hfs#479] [General]
Greetings!
In this version everything is broken in admin/logs
problems with display in
GeoIP in the Country field
Notes
Haven't looked any further yet.

PS if I roll back to v.0.52.0-alpha3? then everything is fine.
Ещё:
![Снимок экрана 2024-02-13 001549](https://github.com/rejetto/hfs/assets/154645533/50d3dc69-ec1d-4092-b3e3-bb5bab5530ff)

**Comment by rejetto** [comment]
thanks for reporting

**Comment by PH5HP** [comment]
Hi,

Tried to find this bug, but I have no problems it seems.

Kind regards,
Henk - https://fileserver.ph5hp.nl

![image](https://github.com/rejetto/hfs/assets/140949730/68758cf2-7e52-465c-9f37-614e69705d73)

**Comment by rejetto** [comment]
I too was unable to see the same "uptime" problem. 
While, I had that problem just before beta4, but it was supposed to be fixed.
@ZIKVLD where do you use HFS, Windows, npx, or other?


i managed to fix the logs problem, instead

**Reply by ZIKVLD** [reply]
Windows

PS Rolled back, now it's like that
![Снимок экрана 2024-02-13 191251](https://github.com/rejetto/hfs/assets/154645533/a0d5fa9a-4cda-4087-b2bf-b9a82db815c6)

**Reply by rejetto** [reply]
you can right-click on check-for-updates, and paste this link
https://github.com/rejetto/hfs/files/14258640/hfs-windows.zip

please let me know if it is good for you

**Reply by ZIKVLD** [reply]
Yes it works
![Снимок экрана 2024-02-13 201607](https://github.com/rejetto/hfs/assets/154645533/dba7148b-a6da-4a36-a9a6-9995eaea3b12)

**Comment by ZIKVLD** [comment]
The following picture is observed.
In full screen version too.
https://github.com/rejetto/hfs/assets/154645533/83ecfd35-3263-4c5d-8983-4e2d852681e7

PS 35" 4K

**Reply by rejetto** [reply]
what browser is this

**Reply by ZIKVLD** [reply]
Microsoft Edge
